Add `Touhou Project Mania Cup 4: Registrations are Now Open!` news post (#12576)
ppy/osu-wiki: news/2024/2024-12-22-touhou-project-mania-cup-4-registrations-now-open.md

---
layout: post
title: "Touhou Project Mania Cup 4: Registrations are Now Open!"
date: 2024-12-22 12:00:00 +0000
---

![](/wiki/Tournaments/TMC/4th/image/banner@2x.png)

With the celebration of the Touhou Project's 30th birthday, the Touhou Project Mania Cup is returning with its 4th iteration, bringing tons of improvement in competitiveness and player enjoyment!

> The [Touhou Project Mania Cup 4th](/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4k tournament. The tournament features 100% Touhou-related music and aims to bring top-tier competition to the tournament scene.

Reviewer: ```suggestion
The [Touhou Project Mania Cup 4th](/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4K tournament exclusively featuring Touhou-related songs. This tournament aims to provide top-tier competition to the tournament scene.
```

> The [Touhou Project Mania Cup 4th](/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4k tournament. The tournament features 100% Touhou-related music and aims to bring top-tier competition to the tournament scene.

Reviewer 2: ```suggestion
The [Touhou Project Mania Cup 4th](https://osu.ppy.sh/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4k tournament. The tournament features 100% Touhou-related music and aims to bring top-tier competition to the tournament scene.
```

> The [Touhou Project Mania Cup 4th](https://osu.ppy.sh/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4K tournament exclusively featuring Touhou-related songs. This tournament aims to provide top-tier competition to the tournament scene.

Author: We have using wiki article links on "This year" words on following parts, also fourm will have links refer to wiki

We’ve had 80 teams [last year](/wiki/en/Tournaments/TMC/3rd), with **Bloxy cola** reaching the top with an undefeated performance. [This year](/wiki/en/Tournaments/TMC/4th), with more teams joining the competition, which team will rise to the challenge and become the best? 

## What's new

> - **A Group Stage will debut this year** featuring a 16-group format. This means that we will have **64 teams** in the main event.

Reviewer: ```suggestion
- **A Group Stage will be introduced in this year**, featuring 16 groups of 4 teams each. This means that we will have **64 teams** in the main event.
```

>   - If more than 70 teams play the qualifier, the group stage will use the split double elimination format.

Reviewer: ```suggestion
  - If there are more than 70 teams participating in the Qualifier, the Group Stage will use the split double elimination format.
```

Reviewer: (also you might want to explain a bit what is this "split double elimination format" since i don't think i've heard this term elsewhere myself)

Reviewer: yeah probably it's better to link that as such

`If there are more than 70 teams participating in the Qualifier, the Group Stage will use the [split double elimination format](http://liquipedia.net/osu/User:RushFTK/Touhou_Project_Mania_Cup/4/Format).`

Author: i will link to wiki rules then.

>   - If fewer than 70 teams play the qualifier, the group stage will use the standard double elimination format.

Reviewer: ```suggestion
  - If there are fewer than 70 teams participating in the Qualifier, the Group Stage will use the standard double elimination format.
```

![Groups format](/wiki/Tournaments/TMC/4th/image/group-stage-bracket.png)

  - **Each team will have 2 bans in the group stage**, providing extra strategic options throughout the different weeks.

> - **The "X" Picks will apply for RC and LN slots**, giving players more options to pick harder charts.

Reviewer 3: ```suggestion
- **A special series of "X" picks will apply for RC and LN slots**, giving players more options to pick harder charts.
```

- Two teams from each group will advance, with a total of 32 teams competing in a standard double-elimination bracket for the championship.

> ## Custom charts 

Reviewer: ```suggestion
## Custom maps
```

> With almost **50 custom mappers** featured, this year we are bringing you a **nearly full custom** mappool from beginner to high-end. We hope this will not only enhance our Touhou journey but also contribute to the osu! community by adding more Touhou maps!

Reviewer: ```suggestion
With almost **50 custom mappers** contributing, this year we are bringing you mappools that are **almost entirely custom-made** with maps ranging from the lower end of the difficulty spectrum to the very top. We hope this will not only enhance our Touhou journey, but also contribute to the osu! community by adding more Touhou maps!
```

> You can check out [Supa7onyz](/beatmaps/artists/435)'s brilliant Grand Finals custom tiebreaker songs from last year as an example!

Reviewer 2: ```suggestion
You can check out [Supa7onyz](https://osu.ppy.sh/beatmaps/artists/435)'s brilliant Grand Finals custom tiebreaker songs from last year as an example!
```

> You can check out [Supa7onyz](https://osu.ppy.sh/beatmaps/artists/435)'s brilliant Grand Finals custom tiebreaker songs from last year as an example!

Reviewer 3: ```suggestion
You can check out [Supa7onyz](https://osu.ppy.sh/beatmaps/artists/435)'s brilliant Grand Finals custom tiebreaker songs from last year as an example:
```

<div align="center">

>     <iframe width="95%" style="aspect-ratio: 16 / 9;" src="//player.bilibili.com/player.html?isOutside=true&aid=487714748&bvid=BV1sN411D7RV&cid=1176362169&p=1" frameborder="0" allowfullscreen></iframe>

Reviewer: might need to ping @Walavouchey for this one, i don't know whether osu! supports embed from bilibili or not :(

Reviewer 3: youtube is definitely preferred. i'd personally be ok with this bilibili embed, but only if you can rectify the following

- there's shouldn't be a weird watermark in the video (who is the uploader? are they related to the tournament?)
- the player should not autoplay
- sound should be enabled when you click play
- for the language part we'll need more opinions

the embed seems to have a bunch of unnecessary parameters. this would be a simpler version
```suggestion
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.bilibili.com/player.html?bvid=BV1sN411D7RV" frameborder="0" allowfullscreen></iframe>
```

Author: - the uploader of this video is the host ([`[GB]Sanae`](https://osu.ppy.sh/users/11238501)), through i also perfer to upload this videos on official youtube channel.
- the player is [`[GB]SuddenDeath`](https://osu.ppy.sh/users/6699923)
- It should be click to play
for last part i could do nothing for it.

</div>

## Tournament schedule

| Event | Dates |
| --: | :-- |
| Registration phase | Dec 22, 2024–Jan 12, 2025 (23:59 UTC) |
| Qualifier showcase | Jan 19 (12:00 UTC) |
| Qualifier stage | Jan 20–Jan 28 |
| Group stage Week 1 | Jan 28–Feb 9 |
| Group stage Week 2 | Feb 14–Feb 16 |
| Round of 32 | Feb 21–Feb 23 |
| Round of 16 | Feb 28–Mar 2 |
| Quarterfinals | Mar 7–Mar 9 |
| Semifinals | Mar 14–Mar 16 |
| Finals | Mar 21–Mar 23 |
| Grand Finals | Mar 28–Mar 30 |

## How do I register?

> Click [here](https://docs.google.com/forms/d/1tl-4NeWZIDI10-1Mkja8lT3DHEpIUB_PK2uqAHptVj8/) and fill in the team registration form!

Reviewer: ```suggestion
Simply click [here](https://docs.google.com/forms/d/1tl-4NeWZIDI10-1Mkja8lT3DHEpIUB_PK2uqAHptVj8/) and fill in the team registration form!
```

To participate in this tournament, you may need to ensure that...

- Your team must consist of 2 to 4 players.

> - All team members must not have violated the [osu! community rules](/wiki/en/Rules) in the past 12 months.

Reviewer: ```suggestion
- All team members have not been violating the [osu! community rules](/wiki/en/Rules) in the past 12 months.
```

> - Your team consists of 2-4 players.

Reviewer 3: ```suggestion
- Your team consists of 2–4 players.
```

If you haven't found enough players yet, feel free to use the `#looking-for-team` channel to find a new member and begin your Touhou adventure!

> Also join our [Discord Channel](https://discord.gg/cAr5VHG7WX) and follow our [Liquipedia page](https://liquipedia.net/osu/Touhou_Project_Mania_Cup/4) to track the latest info. Hope to see you joining us, whether you're a player or spectator!

Reviewer 2: ```suggestion
Also join our [Discord server](https://discord.gg/cAr5VHG7WX) and follow our [Liquipedia page](https://liquipedia.net/osu/Touhou_Project_Mania_Cup/4) to track the latest info. Hope to see you joining us, whether you're a player or spectator!
```

> Also join our [Discord server](https://discord.gg/cAr5VHG7WX) and follow our [Liquipedia page](https://liquipedia.net/osu/Touhou_Project_Mania_Cup/4) to track the latest info. If you haven't found enough players yet, feel free to use the `#looking-for-team` channel to find a new member and begin your Touhou adventure!

Reviewer 3: ```suggestion
Also join our [Discord server](https://discord.gg/cAr5VHG7WX) and follow our [Liquipedia](https://liquipedia.net/osu/Touhou_Project_Mania_Cup/4) or [osu! wiki](/wiki/Tournaments/TMC/4th) page to track the latest info. If you haven't found enough players yet, feel free to use the `#looking-for-team` channel to find a new member and begin your Touhou adventure!
```
